tests/baselines/reference/noParameterReassignmentIIFEAnnotated.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/index.js(3,28): error TS8029: JSDoc '@param' tag has name 'rest', but there is no parameter with that name. It would match 'arguments' if it had an array type.
+
+
+==== tests/cases/compiler/index.js (1 errors) ====
+    self.importScripts = (function (importScripts) {
+        /**
+         * @param {...unknown} rest
+                               ~~~~
+!!! error TS8029: JSDoc '@param' tag has name 'rest', but there is no parameter with that name. It would match 'arguments' if it had an array type.

```

Comment:
@sandersn this is a separate issue I think, but our calculation for the type of an annotated jsdoc rest parameter seems wrong. Or at least inconsistent with this error. Namely, we only pull the type from the jsdoc if it has a leading `...`, but then we pull the inner type node and wrap it in an array - producing `unknown[]` here. This error is fired if you don't make the inner type an array yourself, though, meaning the error indicates you only get rest types if you write `...unknown[]`, which then means the type we assign is actually `unknown[][]`, which is probably never what you wanted.